docs/en/operations/system_tables.md

```diff
@@ -832,30 +832,53 @@ WHERE
 
 If this query doesn’t return anything, it means that everything is fine.
 
-## system.settings {#system-settings}
+## system.settings {#system-tables-system-settings}
 
-Contains information about settings that are currently in use.
-I.e. used for executing the query you are using to read from the system.settings table.
+Contains information about session settings for current user.  
 
 Columns:
 
--   `name` (String) — Setting name.
--   `value` (String) — Setting value.
--   `description` (String) — Setting description.
--   `type` (String) — Setting type (implementation specific string value).
--   `changed` (UInt8) — Whether the setting was explicitly defined in the config or explicitly changed.
--   `min` (Nullable(String)) — Get minimum allowed value (if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings)).
--   `max` (Nullable(String)) — Get maximum allowed value (if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings)).
--   `readonly` (UInt8) — Can user change this setting (for more info, look into [constraints](settings/constraints_on_settings.md#constraints-on-settings)).
+- `name` ([String](../data_types/string.md)) — Setting name.
+- `value` ([String](../data_types/string.md)) — Setting value.
+- `changed` ([UInt8](../data_types/int_uint.md#uint-ranges)) — Shows whether a setting is changed during the session.
+- `description` ([String](../data_types/string.md)) — Short setting description. 
+- `min` ([Nullable](../data_types/nullable.md)([String](../data_types/string.md))) — Minimum value of the setting, if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings). If the setting has no minimum value, contains [NULL](../query_language/syntax.md#null-literal). 
+- `max` ([Nullable](../data_types/nullable.md)([String](../data_types/string.md))) — Maximum value of the setting, if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings). If the setting has no maximum value, contains [NULL](../query_language/syntax.md#null-literal). 
+- `readonly` ([UInt8](../data_types/int_uint.md#uint-ranges)) — Shows whether the current user can change the setting:
```

Comment:
In previous version there is a link to find information why this setting is readony. In new one it is lost
`(for more info, look into [constraints](settings/constraints_on_settings.md#constraints-on-settings))`

Reply:
That's not about constraints, there are complicated set of conditions which lead to readonly. There is no proper text in docs.